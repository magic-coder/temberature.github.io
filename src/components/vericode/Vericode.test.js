import React from "react";
import renderer from "react-test-renderer";
import Vericode from "./Vericode";

it("renders correctly", function() {
  const tree = renderer
    .create(
      <Vericode />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
