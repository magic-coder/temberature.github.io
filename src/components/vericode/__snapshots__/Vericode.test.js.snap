// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  aria-disabled={false}
  className="am-button"
  onClick={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchCancel={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  role="button"
>
  <span>
    获取验证码
  </span>
</a>
`;
